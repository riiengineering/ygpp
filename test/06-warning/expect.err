test/06-warning/input:2: warning: This is a warning
