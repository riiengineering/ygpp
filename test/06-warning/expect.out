Some text before the error occurs.
More text after the warning.
